### PR TITLE
A11y fixes

### PR DESCRIPTION
**Pull request by EdwinGuzman**
This fixes:

* #123 specifying background/foreground colors
* #124 using rem for font size
* #125 updating the "LEARN MORE" button to "VIEW SUMMARY"

I may have missed a few here and there but this is a bulk step into updating everything.

### CODE REVIEW

**Comment by aslagle** [comment]
On iPhone 5, you get 
screen shot 2017-05-02 at 4 22 44 pm

**Comment by EdwinGuzman** [comment]
The above should be fixed. Thanks @aslagle 

Added font sizes and foreground/background colors but need to open another ticket for a normalize css file @ricardoom .

**Comment by ricardoom** [comment]
issue created: #149

File: index.scss [hunk 1]
```diff
@@ -4,17 +4,20 @@ html, body {
   margin: 0;
   padding: 0;
   font-family: Helvetica, sans-serif;
-  font-size: 12px;
+  font-size: 16px;
+  font-size: 1rem;
```
Comment (ricardoom) [inline index.scss]: do we need this? once you set the initial size of 16px, all rules will calculate from this base value.

Comment (EdwinGuzman) [inline index.scss]: Cynthiasays still threw errors when that font-size was not there.

Comment (ricardoom) [inline index.scss]: i'm still not convinced its needed. but I'll approve and we can focus on that w/ our _normalize_

**Pull request merged**